src/ResourceManager/Common/Commands.Common.Strategies.UnitTest/TimeSlotTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Common.Strategies.UnitTest
+{
+    public class TimeSlotTest
+    {
+        [Fact]
+        public void AddTest()
```

Comment:
This should be a check-in tess, we use traits to select tests used for check-in, see any of the test projects 
```[Trait(Category.AcceptanceType, Category.CheckIn)]```

Reply:
fixed.